### PR TITLE
Integration test

### LINKED ISSUE #110: Create github pipeline that executes intergration tests for reef-explorer
We should create a github pipeline that executes integration tests for reef-explorer.

### PR DESCRIPTION
**Pull request by frisitano**
Work in progress

closes: #110

### CODE REVIEW

**Comment by frisitano** [comment]
Currently inefficient as node image is built from source every time.  I will create an image and publish to docker hub.  See: https://github.com/reef-defi/reef-chain/issues/58

**Comment by frisitano** [comment]
Node image now available on docker-hub at `taulepton/reef:v8`.  This has reduced "build" time from 47m to 10m.  Total time reduced from 1h to 22m.

File: api/src/index.ts [hunk 1]
```diff
@@ -79,7 +79,3 @@ const errorHandler = (err: Error, req: Request, res: Response) => {
 
 app.use(errorHandler);
 
-app.listen(config.httpPort, async () => {
```
Comment (Frenkiee) [inline api/src/index.ts:82]: `app.listen`is needed in express framework

Comment (frisitano) [reply api/src/index.ts:82]: yeah this is a work in progress.  Had to remove it to run some tests but will add it back shortly.

Comment (frisitano) [reply api/src/index.ts:82]: Just to give some context. I want to force the integration tests to fail hence removing the `app.listen` was a simple way to achieve this.

**Pull request merged**